src/vm/eventpipeeventinstance.cpp

```diff
@@ -53,7 +53,15 @@ EventPipeEventInstance::EventPipeEventInstance(
 
     m_pData = pData;
     m_dataLength = length;
-    QueryPerformanceCounter(&m_timeStamp);
+
+    // Declare the timeStamp as a local and require that it be 4-byte aligned.
+    // If the timeStamp is not 4-byte aligned, QueryPerformanceCounter will return E_NOACCESS on some versions of Windows.
+    // This codepath is more susceptible to having an unaligned m_timeStamp because EventPipeEventInstance
+    // is initialized via placement-new into the EventPipe circular buffer.
+    DECLSPEC_ALIGN(4) LARGE_INTEGER timeStamp;
```

Comment:
You do not need ` DECLSPEC_ALIGN` here. The compiler will guarantee sufficient alignment by default. (And that alignment is 8 on some platforms.)

Reply:
Thanks @jkotas.  I was just investigating this to make sure that we wouldn't need a fix for other call sites in the runtime.  I'm glad to see that we don't.  I've pushed a change to remove ```DECLSPEC_ALIGN```.